code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -76,6 +76,10 @@
 
 	RegisterSignal(src, COMSIG_MOB_CLICK_ALT, .proc/send_order)
 	RegisterSignal(src, COMSIG_ORDER_SELECTED, .proc/set_order)
+	RegisterSignal(SSdcs, COMSIG_GLOB_OB_LASER_SENT, .proc/receive_laser_OB)
+	RegisterSignal(SSdcs, COMSIG_GLOB_CAS_LASER_SENT, .proc/receive_laser_CAS)
```

Comment:
Rename these signals to end with _CREATED instead of _SENT as that is more indicative

Reply:
Applied